tools/rpkg/tests/testthat/test_relational.R

```diff
@@ -336,3 +336,15 @@ test_that("R semantics for arithmetics sum function are respected", {
    expect_true(is.na(res[[1]]))
 })
 
+
+test_that("anti joins for neq_na_matches works", {
+   dbExecute(con, 'CREATE MACRO "___eq_na_matches_na"(a, b) AS ((a IS NULL AND b IS NULL) OR (a = b))')
```

Comment:
To avoid future head scratching:

```suggestion
   dbExecute(con, 'CREATE OR REPLACE MACRO "___eq_na_matches_na"(a, b) AS ((a IS NULL AND b IS NULL) OR (a = b))')
```